forms/api/form_info.php

```
@@ -0,0 +1,19 @@
+}
+
+try {
+    $form = fetch_active_form();
+    $addons = fetch_addons_from_form_id($form['id']);
+
+    respond([
+        'form' => $form,
+        'addons' => $addons,
+    ]);
+} catch (Throwable $e) {
+    error_log($e->getMessage() ?? "No hay curso activo");
```

PHPStan (2.1.46)

[warning] Expression on left side of ?? is not nullable.


(nullCoalesce.expr)